apps/api/docker-compose.yml

```diff
@@ -3,6 +3,7 @@ name: pcgl-daco-database
 services:
   postgres:
     image: postgres:15-alpine
+    container_name: pcgl-daco/postgres
```

Comment:
Good call to add the container_name but attempting to start the container throws an error:
`Error response from daemon: Invalid container name (c18b95f7c79a_pcgl-daco/postgres), only [a-zA-Z0-9][a-zA-Z0-9_.-] are allowed`
You'll have to replace the `/`

Reply:
oops, thanks.